scripts/qa_crowbarsetup.sh

```diff
@@ -2141,6 +2141,13 @@ function onadmin_allocate()
         done
     fi
 
+    # set rootfs for all nodes when want_rootfs is set
+    if [ -n "$want_rootfs" ] ; then
+        for node in `get_all_discovered_nodes` ; do
+            set_node_fs $node "$want_rootfs"
```

Comment:
and now the docker case above can be reduced to [ -n "$want_docker" ] && want_rootfs=btrfs